src/recognition/recognition.service.ts

```diff
@@ -20,33 +20,22 @@ export class RecognitionService {
         private recognitionsRepository: Repository<Recognition>,
     ){}
 
-    
+    async findCompRec(id: number): Promise<Recognition[]>{
+     return await this.recognitionsRepository.find({where:{companyCompanyId:id}});
+    }
+
     async findAll(): Promise<Recognition[]>{
-        return this.recognitionsRepository.find();
+        return await this.recognitionsRepository.find();
      }
 
-    async createRec(recDto: CreateRecDto): Promise<Recognition> {
-        const rec = new Recognition();
-        rec.msg = recDto.msg;
-        rec.postDate = new Date();
-        rec.company = await this.companyRepository.findOne({where:{companyId : recDto.company}});
-        rec.empFrom = await this.userRepository.findOne({where:{employeeId:recDto.employeeFrom}});
-        rec.empTo = await this.userRepository.findOne({where:{employeeId:recDto.employeeTo}});
-        rec.tags = [];
-        if(recDto.tags != undefined){
-        for(let i = 0; i < recDto.tags.length;i++){
-            const tag = await this.tagRepository.findOne({ where: { tagId: recDto.tags[i] } });
-            if(tag != undefined){
-            rec.tags.push()
-            }
-        }
-    }
-        await this.recognitionsRepository.save(rec);
-        return rec
+    async createRec(recognition: Recognition): Promise<Recognition> {
```

Comment:
Yeah I think that's definitely the best plan

Reply:
I'm not sure if this can be done in github, but would it be possible to only commit the changes with the /all/company endpoint without committing the changes to the create/delete rec data types?

Reply:
I'm not sure but @smalhotra3599 probably would be able to answer that. Neither of the changes are things we are demoing this week so I think we should be fine just holding off merging until after

Reply:
Ok, sounds good